src/components/DataBar/DataBar.stories.tsx

```diff
@@ -19,18 +19,11 @@ export default {
   },
   parameters: {
     badges: ['1.0'],
+    backgrounds: {
```

Comment:
The background parameter lets you change the default background color on individual stories. This is better than doing the same using a special decorator with inline styles.

Added to both databar and progress bar to align snapshots

Reply:
Wow, pretty cool

Reply:
the values we can use are defined here: https://github.com/chanzuckerberg/edu-design-system/blob/main/.storybook/preview.tsx#L48-L63